ep/build-arrow/src/build_arrow_for_velox.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -exu
+
+#setting gluten root path
+GLUTEN_DIR="/gluten"
+ARROW_ROOT="$GLUTEN_DIR/ep/build-arrow/build/arrow_install"
+
+NPROC=$(nproc --ignore=2)
+TESTS=OFF
+BENCHMARKS=OFF
+ENABLE_EP_CACHE=OFF
+BUILD_TYPE=RelWithDebInfo
+
+echo "CMAKE Arguments:"
+echo "TESTS=${TESTS}"
+echo "ARROW_ROOT=${ARROW_ROOT}"
```

Comment:
useless

Reply:
Thank you, I am going to delete after I finish debugging.